opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationResult.java

```diff
@@ -125,25 +125,42 @@ public SELF open_banking_can_read_anton_brueckner_account_data_using_consent_bou
         return open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session(true);
     }
 
+    @SneakyThrows
+    public SELF open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session(String bankProfileId) {
+        return open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session(true, bankProfileId);
+    }
+
+    @SneakyThrows
+    public SELF open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session(
+            boolean validateResourceId
+    ) {
+        open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session_and_bank_profile_id(validateResourceId, SANDBOX_BANK_PROFILE_ID);
+        return self();
+    }
+
     @SneakyThrows
     public SELF open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session(
-        boolean validateResourceId
+            boolean validateResourceId, String bankProfileId
     ) {
-        ExtractableResponse<Response> response = withAccountsHeaders(ANTON_BRUECKNER)
+        open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session_and_bank_profile_id(validateResourceId, bankProfileId);
+        return self();
+    }
+
+    private void open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session_and_bank_profile_id(boolean validateResourceId, String bankProfileId) {
```

Comment:
Use 
```
        ExtractableResponse<Response> response = withAccountsHeaders(ANTON_BRUECKNER, bankProfileId)
                    .header(SERVICE_SESSION_ID, serviceSessionId)
                .when()
                    .get(AIS_ACCOUNTS_ENDPOINT)
                .then()
                    .statusCode(HttpStatus.OK.value())
                    .body("accounts[0].iban", equalTo(ANTON_BRUECKNER_IBAN))
                    .body("accounts[0].resourceId", validateResourceId ? equalTo("cmD4EYZeTkkhxRuIV1diKA") : instanceOf(String.class))
                    .body("accounts[0].currency", equalTo("EUR"))
                    .body("accounts[0].name", equalTo("anton.brueckner"))
                    .body("accounts", hasSize(1))
                .extract();
```
formatting everywhere (it is cleared in this PR), as it improves readability